engine_memory_test.go

```diff
@@ -400,9 +400,9 @@ var recoverTests = []struct {
 	HistorySize     int
 	HistoryLifetime int
 	NumPublications int
-	SinceSeq        uint32
 	NumRecovered    int
 	Sleep           int
+	SinceSeq        uint32
```

Comment:
This change have broken tests since order is important below.

Reply:
oops, I'll keep it on mind ;)